docs/technical_reference/unit_models/treatment_models/med_md_semibatch.rst

```diff
@@ -0,0 +1,156 @@
+
+   "Temperature from S2", ":math:`T_{S2}`", "Feed temperature to MD", ":math:`T_{in,MD}`"
+   "TDS mass flow rate from S2 to MD", ":math:`m_{TDS,to-MD}`", "TDS mass flow rate fed to MD", ":math:`m_{TDS,MD}`"
+   "H2O mass flow rate from S2 to MD", ":math:`m_{H2O,to-MD}`", "H2O mass flow rate fed to MD", ":math:`m_{H2O,MD}`"
+   "Flow pressure from S2", ":math:`P_{S2}`", "Feed pressure to VAGMD", ":math:`P_{in,MD}`"
+   "Temperature from S2", ":math:`T_{S2}`", "Temperature of liquid remained in M1", ":math:`T_{in,MD}`"
+   "TDS mass flow rate from S2 and remained in M1", ":math:`m_{S2,remained}`", "TDS mass flow rate remained in M1", ":math:`m_{TDS,remained}`"
+   "H2O mass flow rate from S2 and remained in M1", ":math:`m_{S2,remained}`", "H2O mass flow rate remained in M1", ":math:`m_{H2O,remained}`"
+   "Flow pressure from S2", ":math:`P_{S2}`", "Pressure of liquid remained in M1", ":math:`P_{remained}`"
+   "Liquid volume in tank", ":math:`V_{tank}`", "Tank volume from previous step", ":math:`V_{tank,pre}`"
+
+These variables will be unfixed in each period after connected to the previous step to maintain the degree of freedom.
+
+Model details
+-------------
+
+* Unit model integrated in the flowsheet
+  
+For the technical details of LT-MED and VAGMD model, please refer to their documentation in the REFLO package.
+
+For the Mixer and Separator unit model, please refer to the documentation of IDAES package.
+
+* Initial condition at the first period
+
+VAGMD feed temperature and concentration are the same as MED brine at the beginning of the processing progress, 
+and so is the mixed liquid in M1, assuming the initial volume in M1 is the user specified batch volume.
+The MED brine tank starts as empty.
+
+* Discharging and refilling phase 
+
+The length of discharging and refilling phase is determined by the following equation:
+
+:math:`T_{reflling} = \frac{V_{batch} - V_{tank_stored}}{\nu_{MED,brine} - \nu_{MD,distillate}}`
+
+where :math:`\nu_{MED,brine}` and :math:`\nu_{MD,distillate}` are the volumetric flow rate of MED brine and MD distillate
+
+* System scaling
+
+The system can be scaled for cost analysis purpose. The user needs to specify the target system capacity
+``target_system_capacity``, and the model will calculatesthe capacity of MED and MD respectively, based on the 
+performance of a specified semibatch operation.
+
+
+Methods
+-------
+
+This unit model class includes the following methods for functionalities. 
+
+``add_costing_packages``: Creates a costing block and setup the system based on the target capacity.
+
+``get_model_performance``: Returns the overall performance of the batch operation in a dictionary, and the timewise system performance of the processing phase in a pandas dataframe.
+
+``get_costing_performance``: Returns the cost performance of a system with a target capacity scaled upon the specified MED-MD module in the unit model.
+
+
+Equations
```

Comment:
Equations section should follow the variables section

Reply:
Section order adjusted.